azext_edge/edge/providers/orchestration/upgrade2.py

```diff
@@ -0,0 +1,309 @@
+# coding=utf-8
```

Comment:
if you delete the og file - may as well just rename this to upgrade.py

Reply:
if you are doing this for the git compare - you can do it as a last push before merge

Reply:
Yes this is to simplify the review.

Reply:
Going to do in subsequent PR.